deploy-service/common/src/main/java/com/pinterest/deployservice/bean/DeployConstraintBean.java

```diff
@@ -9,6 +9,7 @@
  * constraint_id     VARCHAR(22)         NOT NULL,
  * constraint_key    VARCHAR(64)         NOT NULL
  * max_parallel      BIGINT              NOT NULL,
+ * inclusive         TINYINT(1)          NOT NULL DEFAULT 0,
```

Comment:
instead of support yet another special constraint, I wonder if we can come up with a more generic solution.

Reply:
good point, just updated into an enum field